recipes/sfdc-merlion/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "sfdc-merlion" %}
```

Comment:
Looking [here](https://github.com/salesforce/Merlion) and [searching on PyPI](https://pypi.org/search/?q=sfdc-merlion) should this recipe instead be for "salesforce-merlion"?

Reply:
they just changed it recently it seems